### ISSUE TITLE
Blade of Hoarfrost Shard Weapon Not Rendering Properly

### ISSUE THREAD

**Issue by PHILLIPGATLIN** [MSRevive/MSCScripts#131]
The blade of the Hoarfrost Shard weapon is suppose to be translucent but in some situations the blade is completely invisible, such as when the blade is over a transparent map texture.

**Comment by greatguys1** [comment]
I dont think this is a script issue, but I dont think we can fix it either- it might just be an engine issue. For now I'll keep this issue open and over here until we know exactly where the issue comes from.
